src/main.cpp

```diff
@@ -257,6 +257,9 @@ static void set_allowed_options(OptionList *allowed_options)
 			_("Set world by name (implies local game)"))));
 	allowed_options->insert(std::make_pair("quiet", ValueSpec(VALUETYPE_FLAG,
 			_("Print to console errors only"))));
+	allowed_options->insert(std::make_pair("color", ValueSpec(VALUETYPE_STRING,
```

Comment:
maybe the preprocessor could be used to not include the `auto` option in windows builds if it doesn't work?

Reply:
Nice idea. I didn't exclude it everywhere because at some point in time